### DISCUSSION TITLE
User in new position with each refresh?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#396] [Q&A] [answered]
Hello, I was wondering if there was a way to stop a user from ending up in new position within the networked-aframe? Everytime I need to refresh the page I ended up in a few position.

Joseph

**Comment by vincentfretin** [comment, marked as answer]
Hello, the random position is done by the `spawn-in-circle` component. If you don't want this behavior, just remove the component.

**Reply by Therealjosephchrzempiec** [reply]
Thank you very much that worked. I deleted itand I'm where I needed to be.
